src/AuthLDAP.php

```diff
@@ -3010,22 +3026,48 @@ public static function connectToServer(
 
         $ds = @ldap_connect($host, intval($port));
         if ($ds) {
```

Comment:
I would probably be easier to read if error/return was triggered at this point on failure. If would prevent to have to encapsulate the whole logic in a `if ($ds) { ... }`

```php
$ds = @ldap_connect($host, intval($port));
if ($ds === false) {
    trigger_error(
        sprintf(
            "Unable to connect to LDAP server %s:%s",
            $host,
            $port
        )
    );
    return false;
}
...
```